src/test/java/com/example/employees/service/EmployeesServiceTest.java

```diff
@@ -0,0 +1,165 @@
+package com.example.employees.service;
+
+import com.example.employees.controller.EmployeeController;
+import com.example.employees.controller.EmployeesController;
+import com.example.employees.model.Employee;
+import com.example.employees.model.Gender;
+import com.example.employees.repo.EmployeeRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.domain.Page;
+
+import java.time.LocalDate;
+
+@SpringBootTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class EmployeesServiceTest {
```

Comment:
minor: extra line at the beginning of some functions
and one extra in the end of `shouldGetPageWithTwoEmployees`